### PR TITLE
Update minishift.rb

### PR DESCRIPTION
**Pull request by asm0dey**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/minishift.rb [hunk 1]
```diff
@@ -8,5 +8,5 @@
   name 'Minishift'
   homepage 'https://github.com/minishift/minishift'
 
-  binary 'minishift-darwin-amd64', target: 'minishift'
+  binary 'minishift-1.0.0-rc.2-darwin-amd64.tgz', target: 'minishift'
```
Comment (vitorgalvao) [inline Casks/minishift.rb]: You’re not linking a compressed file as a binary. That makes no sense. Please leave this as was.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]
It’s correct as is.

**Comment by asm0dey** [comment]
I think this is wrong repo because rc isn't stable, I think…

**Comment by vitorgalvao** [comment]
That can be considered stable.

**Pull request merged**